pkgs/by-name/fc/fcitx5-material-color/package.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+}:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
Here is a little suggestion, to use `finalAttrs` pattern instead.

+ [Historical Discussion](https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293)
+ [Documentation](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes)
+ [Implementation Example](https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe)

Reply:
learned new things, thx